### PR TITLE
Add ability to export Positional weights in fakely quant onnx

### PR DESCRIPTION
**Pull request by lapid92**
## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/exporter/model_exporter/pytorch/base_pytorch_exporter.py [hunk 1]
```diff
@@ -73,8 +73,22 @@ def _set_quantized_weights_in_wrapper(layer:PytorchQuantizationWrapper):
...
+            # Replace it with the quantized version as a new parameter
+            setattr(linear_layer, name, torch.nn.Parameter(quantized_weight))
+        else:
+            # If the name is not a string, it must be an integer representing a positional weight
+            assert isinstance(name, int)
+            attr_name = f'{QUANTIZED_POSITIONAL_WEIGHT}_{name}'
+            if hasattr(layer, attr_name):
+                delattr(layer, attr_name)
+
+            # Add the quantized weight as a new attribute directly on the parent layer
```
Comment (elad-c) [inline model_compression_toolkit/exporter/model_exporter/pytorch/base_pytorch_exporter.py:95]: explain we do that so the wrapper forward will work. that's why we select that specific attribute name.

File: model_compression_toolkit/exporter/model_exporter/pytorch/base_pytorch_exporter.py [hunk 2]
```diff
@@ -73,8 +73,22 @@ def _set_quantized_weights_in_wrapper(layer:PytorchQuantizationWrapper):
...
-        setattr(linear_layer, name, torch.nn.Parameter(quantized_weight))
+
+        # If the name is a string, we assume it's a named attribute of the linear layer
+        if isinstance(name, str):
+            # Remove the existing attribute from the linear layer
+            delattr(linear_layer, name)
+            # Replace it with the quantized version as a new parameter
+            setattr(linear_layer, name, torch.nn.Parameter(quantized_weight))
+        else:
+            # If the name is not a string, it must be an integer representing a positional weight
```
Comment (elad-c) [inline model_compression_toolkit/exporter/model_exporter/pytorch/base_pytorch_exporter.py:84]: and the int is the position in the later arguments.

**Review by elad-c — APPROVED** [review]
